src/tests/multihost/sssd/testlib/common/utils.py

```diff
@@ -2025,31 +2025,45 @@ def get_zones(self):
 
     def print_zone(self, zone):
         """ Prints all the contents of a zone file, takes domain.com or 1.168.192.in-addr.arpa string """
-        zone = self.ad_host.run_command(f'dnscmd.exe /zoneprint {zone} | grep -v \; | sed "s/\t/ /g" | grep e |'
-                                        f'sed "s/\   / /g"')
+        zone = self.ad_host.run_command(f'dnscmd.exe /zoneprint {zone}')
+#        zone = self.ad_host.run_command(f'dnscmd.exe /zoneprint {zone} | grep -v \; | sed "s/\t/ /g" | grep e |'
```

Comment:
Please remove the commented-out code .
Make sure that python does not complain about " DeprecationWarning: invalid escape sequence '\;'".

Reply:
You can also use raw f-strings where/when absolutely necessary.

Reply:
I forgot I commented that out for debugging.